python_modules/libraries/dagster-dg/dagster_dg/templates/build-location-fragment.yaml

```diff
@@ -0,0 +1,16 @@
+- name: Build and upload Docker image for "LOCATION_NAME"
+  if: steps.prerun.outputs.result != 'skip'
+  uses: docker/build-push-action@v4
+  with:
+    context: LOCATION_PATH
+    push: true
+    tags: ${{ env.IMAGE_REGISTRY }}:${{ env.IMAGE_TAG }}-LOCATION_NAME
+    cache-from: type=gha
+    cache-to: type=gha,mode=max
+
+- name: Update build session with image tag for LOCATION_NAME
+  id: ci-set-build-output-example-location
+  if: steps.prerun.outputs.result != 'skip'
+  uses: dagster-io/dagster-cloud-action/actions/utils/dagster-cloud-cli@v0.1
```

Comment:
at some point fairly soon we will likely template in a specific version rather than v0.1 (to start, probably some other constant version defined in code based on what @mlarose gives us - eventually, maybe the version that matches the dg version?) - we could add that templating now or later

Reply:
How about I start by creating a long-running branch called dg-alpha or something like that and use the branch name as a target? so we can commit update without too much manual labor?

Reply:
Anything that lets us experiment with a PEX to start is fine by me. Keep in mind that real users may be using this as soon as next week though, so it won't be alpha for long